### PR TITLE
Add CUDA 11.8

### PR DESCRIPTION
**Pull request by m3vaz**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by m3vaz** [comment]
@conda-forge-admin, please rerender

File: recipe/install_cuda.bat [hunk 1]
```diff
@@ -24,6 +24,7 @@ if "%CUDA_VERSION%" == "11.4" goto cuda114
 if "%CUDA_VERSION%" == "11.5" goto cuda1151
 if "%CUDA_VERSION%" == "11.6" goto cuda116
 if "%CUDA_VERSION%" == "11.7" goto cuda117
+if "%CUDA_VERSION%" == "11.8" goto cuda117
```
Comment (jaimergp) [inline recipe/install_cuda.bat:27]:
```suggestion
if "%CUDA_VERSION%" == "11.8" goto cuda118
```

File: recipe/install_cuda.bat [hunk 2]
```diff
@@ -169,6 +170,14 @@ set "CUDA_INSTALLER_CHECKSUM=a2388d0044b2dd6a3469938eb6108c85"
 set "CUDA_COMPONENTS=%CUDA_COMPONENTS% nsight_nvtx_%CUDA_VERSION%"
 goto cuda_common
 
+:cuda117
```
Comment (jaimergp) [inline recipe/install_cuda.bat:173]:
```suggestion
:cuda118
```

**Review by kkraus14 — APPROVED** [review]

**Review by jaimergp — APPROVED** [review]
LGTM. I'd wait before merge given the current Azure incidence.

**Review by beckermr — CHANGES_REQUESTED** [review]
Please do not merge this pr yet.

**Comment by m3vaz** [comment]
@beckermr Can we merge this yet?

**Comment by beckermr** [comment]
No do not merge.

**Comment by m3vaz** [comment]
From internal discussion, closing until further notice.

**Pull request closed without merging**